Changes in module 'nf-core/vcf2maf'
--- modules/nf-core/vcf2maf/main.nf
+++ modules/nf-core/vcf2maf/main.nf
@@ -10,8 +10,11 @@
         'biocontainers/mulled-v2-b6fc09bed47d0dc4d8384ce9e04af5806f2cc91b:305092c6f8420acd17377d2cc8b96e1c3ccb7d26-0' }"
 
     input:
-    tuple val(meta), path(vcf) // Use an uncompressed VCF file!
+    tuple val(meta), path(vcf) // Use an uncompressed VCF file!    
+    val(normal_id)             // Optional
+    val(tumor_id)              // Optional
     path fasta                 // Required
+    val genome                 // Required
     path vep_cache             // Required for VEP running. A default of /.vep is supplied.
 
     output:
@@ -25,6 +28,10 @@
     def args          = task.ext.args   ?: ''
     def prefix        = task.ext.prefix ?: "${meta.id}"
     def vep_cache_cmd = vep_cache       ? "--vep-data $vep_cache" : ""
+    def genome_build  = genome          ? "--ncbi-build $genome"  : "" 
+    def normal        = normal_id       ? "--normal-id $normal_id --vcf-normal-id $normal_id" : ""
+    def tumor         = tumor_id        ? "--tumor-id $tumor_id   --vcf-tumor-id  $tumor_id"  : ""
+
     // If VEP is present, it will find it and add it to commands.
     // If VEP is not present they will be blank
     def VERSION = '1.6.21' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
@@ -42,7 +49,10 @@
         $args \\
         \$VEP_CMD \\
         $vep_cache_cmd \\
+        $genome_build \\
         --ref-fasta $fasta \\
+        $normal \\
+        $tumor  \\
         --input-vcf $vcf \\
         --output-maf ${prefix}.maf
 

************************************************************
